src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -0,0 +1,108 @@
+                                    (reset! selected [])
+                                    (rf/dispatch [:bottom-sheet/hide]))
+            :accessibility-label :confirm-selection}
+           (i18n/label :t/confirm-selection)]])))])
+
+(defn clear-button
+  []
+  (when (pos? (count @selected))
+    [rn/touchable-opacity
+     {:on-press            #(reset! selected [])
+      :style               (style/clear-container)
+      :accessibility-label :clear}
+     [quo2/text {:weight :medium} (i18n/label :t/clear)]]))
+
+(defn image
+  [item index _ {:keys [window-width]}]
+  [rn/touchable-opacity
+   {:active-opacity      1
+    :on-press            (fn []
+                           (if (some #{item} @selected)
+                             (reset! selected (vec (remove #(= % item) @selected)))
+                             (swap! selected conj item)))
+    :accessibility-label (str "image-" index)}
+   [rn/image
+    {:source {:uri item}
+     :style  (style/image window-width index)}]
+   (when (some #{item} @selected)
+     [rn/view {:style (style/overlay window-width)}])
+   (when (some #{item} @selected)
+     [info-count/info-count
+      {:style               style/image-count
+       :accessibility-label (str "count-" index)}
+      (inc (utils/first-index #(= item %) @selected))])])
+
+(defn photo-selector
+  [chat-id]
+  (rf/dispatch [:chat.ui/camera-roll-get-photos 20])
+  (let [selected-images (keys (get-in (rf/sub [:chat/inputs]) [chat-id :metadata :sending-image]))]
+    (when selected-images
+      (reset! selected (vec selected-images)))
+    [:f>
+     (fn []
+       (let [{window-height :height window-width :width} (rn/use-window-dimensions)
+             safe-area                                   (safe-area/use-safe-area)
+             camera-roll-photos                          (rf/sub [:camera-roll/photos])
+             end-cursor                                  (rf/sub [:camera-roll/end-cursor])
+             loading?                                    (rf/sub [:camera-roll/loading-more])
+             has-next-page?                              (rf/sub [:camera-roll/has-next-page])]
+         [rn/view {:style {:height (- window-height (:top safe-area))}}
+          [rn/touchable-opacity
+           {:on-press #(js/alert "Camera: not implemented")
+            :style    (style/camera-button-container)}
+           [quo2/icon :i/camera {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
+          [rn/view
+           {:style {:flex-direction :row
+                    :position       :absolute
+                    :align-self     :center}}
+           [quo2/text {:weight :medium} (i18n/label :t/recent)]
+           [rn/view {:style (style/chevron-container)}
+            [quo2/icon :i/chevron-down {:color (colors/theme-colors colors/neutral-100 colors/white)}]]]
+          [clear-button]
+          [rn/flat-list
+           {:key-fn                  (fn [item] item)
```

Comment:
You can just use `identity` here.

Reply:
Is this a function that just returns the argument as it is?